playbooks/notifications/email-notify-list-of-users.yml

```diff
@@ -14,9 +17,11 @@
         markdown_content: "{{ body }}"
     - set_fact:
         mail: "{{ mail | combine({ 'subject': title, 'body': md_to_html.html_body_message }) }}"
+
     - set_fact:
-        list_of_mail_to: "{{ list_of_mail_to | default([]) }} + [ '{{ item.email }}' ]"
```

Comment:
The issue is the placement of the `{{` and `}}` - if you change this line to the following it should fix the problem. 
```
list_of_mail_to: "{{ list_of_mail_to | default([]) + [ item.email ] }}"
```

Reply:
Yes, that was the fix.